ENH: Include LAMP motion class (#613)
pcdshub/pcdsdevices: docs/source/upcoming_release_notes/614-LAMP_motion_system_class.rst

614 LAMP motion system class

> #################

Reviewer: Oh, and this annoying thing about sphinx/restructured text (have to be the same length):
```suggestion
############################
```

Author: Ah of course, resolved in commit 5d3e726

API Changes
-----------
- N/A

Features
--------
- N/A

Device Updates
--------------
- N/A

New Devices
-----------

> - LAMP motion Class for the LAMP endstation TMO. This indluces the following motion axes:

Reviewer: ```suggestion
- LAMP motion Class for the LAMP endstation TMO. This includes the following motion axes:
```

Author: Thanks for catching that! Resolved in commit 5d3e726

> - LAMP Class: TMO-LAMP Motion system class:
>  - Gas Jet X/Y/Z Axes
>  - Gas Needle X/Y/Z Axes
>  - Sample Paddle X/Y/Z Axes 

Reviewer 2: Suggesting adding more context in case someone doesn't know that lamp isn't a light you put on your desk. Also some formatting.
```suggestion
- LAMP motion Class for the LAMP endstation in TMO. This includes the following motion axes:
  - Gas Jet X/Y/Z
  - Gas Needle X/Y/Z
  - Sample Paddle X/Y/Z
```

Author: Resolved in commit 461e9d4
